fsi-workflow/03_fluidMesh/tasks.md

```diff
@@ -164,14 +181,18 @@ Notes:
 - the timestep depends on the `deltaT` parameter in the `controlDict` file, but it is not relevant
 - you can obtain the final mesh in the `constant` directory, without the intermediate steps, by adding the `-overwrite` option to `snappyHexMesh`
 
+**TODO:** We probably want to explicitly tell people to use the `-overwrite`, right? Or is that not needed?
```

Comment:
@Ccaccia73 do we need to run `snappyHexMesh -overwrite`, or not?

Reply:
We don't need it. My idea was to let the execution without `-overwrite` so that they can see what each step of snappy does. This is also useful in general to understand where to make changes in your mesh parameters, or re-execute only the subsequent steps (changing from true to false). Of course, when you are confident and just want your final mesh you can add it

Reply:
Addressed in https://github.com/precice/community-training/pull/8